File: B-1368.cpp
#include <bits/stdc++.h>
/*
** Author: wkwkwill (Willy I. K.)
** 2020/06/18
*/
using namespace std;
using ll = long long int;

string ans = "codeforce", s = "s";

int main()
{
    ios_base::sync_with_stdio(false);
    cin.tie(NULL);
    ll k;
    cin >> k;
        int j = 1;
        for (; j <= k; j += j)
        {
            if (j > pow(2, 26)) break;
            if (j >= 2) s += s;
        }
        j /= 2;
        printf("codeforce");
        for (int i = 0; i < k / j; ++i)
        {
            printf("%s", s.c_str());
        }
        for (int i = 0; i < k % j; ++i)
        {
            printf("s");
        }
    
    return 0;
}

File: Codeforces/B-1368.py
#include <bits/stdc++.h>
#
# Author: wkwkwill (Willy I. K.)
# 2020/06/18

ans = "codeforce"
s = "s"

k = int(input())
j = 1;
while (j <= k):
    if (j > 2**24):
        break
    if (j >= 2):
        s += s
    j += j

j /= 2
print("codeforce", end='')
i = 0
while (i < int(k / j)):
    print(s, end='')
    i += 1
i = 0
while (i < int(k % j)):
    print("s", end='')
    i += 1